### ISSUE TITLE
Inference failure after fine-tuning GR00T policy for 100k steps on G1_Dex1 Sim dataset

### ISSUE THREAD

**Issue by peterwang779** [unitreerobotics/unitree_lerobot#81]
I am experiencing issues with evaluating a trained GR00T policy in the Isaac Sim environment. Despite training for 100k steps using the provided dataset, the robot fails to complete the task.

### Environment

- Dataset: unitreerobotics/G1_Dex1_PickPlaceRedBlock_Dataset_Sim

- Robot: Unitree G1 (29-DOF) with Dex1 hand.

- Training Framework: unitree_lerobot

### Training Script
I used the following command for training(Total steps: 100,000)：
`python src/lerobot/scripts/lerobot_train.py \
    --dataset.repo_id=unitreerobotics/G1_Dex1_PickPlaceRedBlock_Dataset_Sim \
    --output_dir=/home/fpc/unitree_lerobot/unitree_lerobot/lerobot/outputs/groot_training \
    --policy.push_to_hub=false \
    --policy.type=groot \
    --policy.tune_diffusion_model=false \
    --batch_size=32 \
    --job_name=groot_training`

### Evaluation Workflow

1. Launch Simulation：
`python sim_main.py --device cuda --enable_cameras --task Isaac-PickPlace-RedBlock-G129-Dex1-Joint --enable_dex1_dds --robot_type g129`

2. Run Inference：
`python unitree_lerobot/eval_robot/eval_g1_sim.py \
    --policy.path=unitree_lerobot/lerobot/outputs/groot_training/checkpoints/100000/pretrained_model \
    --repo_id=unitreerobotics/G1_Dex1_PickPlaceRedBlock_Dataset_Sim \
    --root="" \
    --episodes=0 \
    --frequency=30 \
    --arm="G1_29" \
    --ee="dex1" \
    --visualization=true \
    --save_data=false \
    --task_dir="./data" \
    --max_episodes=1200`

### Expected Behavior
The robot should pick the red block and place it in the target area as shown in the dataset.

### Actual Behavior
The evaluation results are consistently failing. https://github.com/user-attachments/assets/eb17cdac-d557-43bc-9595-dcebbb12c04f

### Questions

1. Is 100k steps sufficient for the GR00T policy on this specific task, or does it require more iterations (e.g., 200k-500k)?

2. Are there any recommended hyperparameters (e.g., learning rate, weight decay, or specific batch_size) for fine-tuning GR00T on the Dex1 Sim dataset?

3. Should --policy.tune_diffusion_model be set to true for better convergence in this pick-and-place scenario?

**Comment by Hahalim2022y** [comment]
I encountered the same issue: the robot didn't make any effective grasping motion. It initially just slightly raised its arm, then stayed in place, making some repeated, very small movements. Has anyone found a solution?

**Comment by dengfenglai624** [comment]
> 我也遇到了同样的问题：机器人无法做出有效的抓取动作。它一开始只是稍微抬起手臂，然后就停在原地，重复做一些非常小的动作。有人找到解决方法了吗？

Have you solved this problem?
My offline inference trained pi0.5 model works normally, but the gripper fails to grasp objects when running inference on the real robot.

**Comment by peterwang779** [comment]
> I encountered the same issue: the robot didn't make any effective grasping motion. It initially just slightly raised its arm, then stayed in place, making some repeated, very small movements. Has anyone found a solution?

I switched from the official dataset to my own collected data for training, and it performed well during simulation tests.

**Comment by Hahalim2022y** [comment]
> > I encountered the same issue: the robot didn't make any effective grasping motion. It initially just slightly raised its arm, then stayed in place, making some repeated, very small movements. Has anyone found a solution?我也遇到了同样的问题：机器人无法做出有效的抓取动作。它一开始只是稍微抬起手臂，然后就停在原地，重复做一些非常小的动作。有人找到解决方法了吗？
> 
> I switched from the official dataset to my own collected data for training, and it performed well during simulation tests.我把训练数据集从官方数据集换成了我自己收集的数据，在模拟测试中表现良好。

Thanks! I solved this problem by modifying the image in eval_g1_sim.py to convert from BGR to RGB.

**Comment by Hahalim2022y** [comment]
> > 我也遇到了同样的问题：机器人无法做出有效的抓取动作。它一开始只是稍微抬起手臂，然后就停在原地，重复做一些非常小的动作。有人找到解决方法了吗？
> 
> Have you solved this problem?这个问题解决了吗？ My offline inference trained pi0.5 model works normally, but the gripper fails to grasp objects when running inference on the real robot.我的离线推理训练的 pi0.5 模型工作正常，但在真实机器人上运行推理时，机械臂无法抓取物体。

I solved this problem by modifying the code in eval_g1_sim.py to convert images from BGR to RGB. Currently, I have only successfully run inference in simulation; I haven't used a real robot.

**Comment by peterwang779** [comment]
Hi @yanght24 

"I believe this is due to an issue with the official dataset. As I mentioned in https://github.com/unitreerobotics/unitree_lerobot/issues/78, the task descriptions in the official dataset do not match the actual behaviors. Consequently, I trained the model using a dataset I collected myself and tested it in the simulation environment, which yielded good results."
